src/tasks/peripherals/task_nfc.c

```diff
@@ -7,7 +14,7 @@
 #include "ruuvi_interface_communication_nfc.h"
 #include "ruuvi_interface_communication_radio.h"
 #include "ruuvi_interface_scheduler.h"
-#include "ruuvi_interface_watchdog.h"
+#include "ruuvi_interface_dog.h"
```

Comment:
does this compile?

Reply:
Argh! I accidentally lost the "watch" , restored it to my fork, but it seems I neglected to re-pull it .